packages/astro/src/cli/install-package.ts

```diff
@@ -1,14 +1,15 @@
 import { createRequire } from 'node:module';
 import boxen from 'boxen';
 import ci from 'ci-info';
-import { bold, cyan, dim, magenta } from 'kleur/colors';
 import { detect, resolveCommand } from 'package-manager-detector';
+import colors from 'picocolors';
 import prompts from 'prompts';
 import yoctoSpinner from 'yocto-spinner';
 import type { Logger } from '../core/logger/core.js';
 import { exec } from './exec.js';
 
 const require = createRequire(import.meta.url);
+const { bold, cyan, dim, magenta } = colors;
```

Comment:
In most modules, I refactored to use `colors.method()` directly, but in a couple like this one where there’s _heavy_ use of the APIs, I kept the destructured approach to minimize code changes and keep usage a bit cleaner.